tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "strict": true,
     "noEmit": true,
     "allowSyntheticDefaultImports": true,
-    "types": ["webpack-env", "app/javascript/declarations"]
+    "types": ["webpack-env", "app/javascript/declarations", "jest"]
```

Comment:
Does this mean we "leak" Jest into our production compile?

Reply:
You don't leak `types` :)

Reply:
@neenjaw instead you want to create a `tsconfig.json` inside the tests folder that **extends** the one in the root, that adds these types, otherwise `jest` types are available in non-test files.

Same for `includes`/`excludes`, you want test files to be able to see (import) app files, but not the other way around.

Reply:
:+1:

Reply:
I think this is what you meant?